src/registrar_agent.rs

```diff
@@ -40,11 +53,8 @@ fn is_empty(buf: &[u8]) -> bool {
 
 #[derive(Debug, Serialize, Deserialize)]
 struct Register<'a> {
-    #[serde(
-        serialize_with = "serialize_as_base64",
-        skip_serializing_if = "is_empty"
```

Comment:
Hah, right. I thought I had made the code check whether `ekcert` is in the json, but instead I made it assume that that's there and just check if that's json `null`.
Thanks for finding and fixing that!